src/references/ens/ENSPublicResolver.json

```diff
@@ -633,26 +633,6 @@
         "stateMutability": "nonpayable",
         "type": "function"
     },
-    {
-        "constant": false,
-        "inputs": [
-            {
-                "internalType": "bytes32",
-                "name": "node",
-                "type": "bytes32"
-            },
-            {
-                "internalType": "address",
-                "name": "a",
-                "type": "address"
-            }
-        ],
-        "name": "setAddr",
-        "outputs": [],
-        "payable": false,
-        "stateMutability": "nonpayable",
-        "type": "function"
-    },
```

Comment:
why this got removed? https://etherscan.io/address/0x4976fb03C32e5B8cfe2b6cCB31c09Ba78EBaBa41#writeContract

Reply:
yeah, ethers doesn't play well with dupe methods, it errors out.